tools/peptideshaker/fasta_cli.xml

```diff
@@ -0,0 +1,60 @@
+<tool id="fasta_cli" name="FastaCLI" version="1.0">
+    <description>
+        Appends decoy sequences to FASTA files
+    </description>
+    <macros>
+        <import>macros_basic.xml</import>
+    </macros>
+    <requirements>
+	<requirement type="package" version="@SEARCHGUI_VERSION@">searchgui</requirement>
+	<requirement type="package" version="3.0">zip</requirement>
+    </requirements>
+    <expand macro="stdio" />
+    <command>
+<![CDATA[
+        #set $bin_dir = "bin"
+
+        mkdir output;
+        cwd=`pwd`;
+
+        echo 'DB: ${input_database.display_name} sequences: ${input_database.metadata.sequences}';
+
+        cp '${input_database}' input_database.fasta;
```

Comment:
Wouldn't linking work as well here?

Reply:
Of course, linting was not working on any tool as macros_basic.xml is used by them all.